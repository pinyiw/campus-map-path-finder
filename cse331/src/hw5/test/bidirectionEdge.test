# Create a graph
CreateGraph g

# Add nodes
AddNode g a
AddNode g b
AddNode g c

# Add edges
AddEdge g a b e1
AddEdge g b a e2
AddEdge g a c e3
AddEdge g c a e4
AddEdge g b c e5
AddEdge g c b e6

# Print the outgoing edges from a, b and c
ListChildren g a
ListChildren g b
ListChildren g c