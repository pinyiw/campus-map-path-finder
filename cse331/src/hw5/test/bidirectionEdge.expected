# Create a graph
created graph g

# Add nodes
added node a to g
added node b to g
added node c to g

# Add edges
added edge e1 from a to b in g
added edge e2 from b to a in g
added edge e3 from a to c in g
added edge e4 from c to a in g
added edge e5 from b to c in g
added edge e6 from c to b in g

# Print the outgoing edges from a, b and c
the children of a in g are: b(e1) c(e3)
the children of b in g are: a(e2) c(e5)
the children of c in g are: a(e4) b(e6)